tests/test_api.py

```diff
@@ -1929,11 +1930,14 @@ def test_hierarchical_hashes_array_slice(self) -> None:
         assert response.status_code == 200
         result = json.loads(response.data)
 
-        assert result["sql"].startswith(
+        val = (
             "SELECT arrayMap((x -> replaceAll(toString(x), '-', '')), "
-            "arraySlice(hierarchical_hashes, 0, 2)) FROM errors_local PREWHERE"
+            "arraySlice(hierarchical_hashes, 0, 2)) FROM %s PREWHERE"
+            % (errors_storage.get_table_writer().get_schema().get_table_name())
```

Comment:
I think this would be a bit easier to read if you defined `table_name` as a variable on the line above then use an f-string here instead of the old % formatting way.

Reply:
Ok will do!